src/credentials/FabricTable.cpp

```diff
@@ -438,16 +438,13 @@ CHIP_ERROR FabricInfo::VerifyCredentials(const ByteSpan & noc, const ByteSpan &
 
 FabricTable::~FabricTable()
 {
-    FabricTableDelegate * delegate = mDelegate;
+    FabricTable::Delegate * delegate = mDelegateListRoot;
     while (delegate)
```

Comment:
What is the point of this loop?  It's a no-op now....

Reply:
Ah, to indicate to delegates that outlive this table that they are not in the list anymore?  That would make sense, yes.